sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/models/ClientEndpointType.java

```diff
@@ -0,0 +1,39 @@
+package com.azure.messaging.webpubsub.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Defines values for ClientEndpointType
+ */
+public enum ClientEndpointType {
+    /** Default WebPubSub Client Endpoint. E.g: <code>wss://exampleHost.com/client/hubs/exampleHub</code> */
+    DEFAULT("default"),
+
+    /** MQTT Client Endpoint. E.g: <code>wss://exampleHost.com/client/mqtt/hubs/exampleHub</code> */
```

Comment:
It should be `/clients` (an 's' prefixed). You may have already noticed that it's a different pattern with the `/client/hubs/hub`